app/ModelSerializers/Summit/Registration/PromoCodes/SummitRegistrationPromoCodeSerializer.php

```diff
@@ -69,6 +70,14 @@ public function serialize($expand = null, array $fields = [], array $relations =
             $values['allowed_ticket_types'] = $ticket_types;
         }
 
+        if(in_array('tags', $relations)) {
```

Comment:
refactor this serializer to use serialization by reflections as we do for 
https://github.com/OpenStackweb/summit-api/blob/7d55ec41e82c2a40ad771cad25580b2492fb89e3/app/ModelSerializers/AbstractMemberSerializer.php#L91

Reply:
done